metadata/client.go

```diff
@@ -1432,6 +1451,17 @@ func (variant *FeatureVariant) LocationColumns() interface{} {
 	return columns
 }
 
+func (variant *FeatureVariant) LocationFunction() interface{} {
+	if variant.Category() != ON_DEMAND_CLIENT {
```

Comment:
minor note, but if one category is PRE_CALCULATED I'd imagine the other would be CLIENT_CALCULATED